docs/zh_CN/dgs/dataloader.md

```diff
@@ -1,3 +1,183 @@
-# 从数据源加载数据
+# Loading From Your Data Source
 
-（comming soon...）
+By using kafka queues as the entry for streaming graph updates, the Dynamic-Graph-Service (abbreviated as dgs) can
+decouple the different data sources and sampling workers.
+Users can develop their own data loaders to ingest the source data and push graph updates into the output kafka queues
+for further consuming.
+
+## Rules to Follow
+
+When processing source data in the dataloader, these rules should be followed:
+- The graph schema from source data must be consistent with dgs.
+- A produced kafka message must be a batch of graph updates, defined as a flatbuffers table [RecordBatchRep](https://github.com/alibaba/graph-learn/blob/master/dynamic_graph_service/fbs/record.fbs).
+- Graph updates are partitioned for sampling workers, all records in one batch must have the same data partition id.
+
+## Dataloader SDK
+
+In practice, it is difficult for users to batch records with dgs graph schema, manage partitioning logic and
+produce partitioned batches into kafka queues with themselves.
+Thus, we provide a dataloader sdk (a c++ lib) to help users to do these things.
+
```

Comment:
The above paragraph:
"We provide a dataloader sdk (a c++ lib) to help users to batch records with following the specified graph schema in DGS, manage partitioning logics and dispatch partitioned records to the kafka queues."

Reply:
done